src/platform/EFR32/efr32-chip-mbedtls-config.h

```diff
@@ -241,8 +242,8 @@ typedef void mbedtls_ecp_restart_ctx;
 #define MBEDTLS_ENTROPY_MAX_SOURCES 1   /**< Maximum number of sources supported */
 
 // #if OPENTHREAD_CONFIG_MULTIPLE_INSTANCE_ENABLE
-#define MBEDTLS_PLATFORM_STD_CALLOC calloc /**< Default allocator to use, can be undefined */
-#define MBEDTLS_PLATFORM_STD_FREE free     /**< Default free to use, can be undefined */
+#define MBEDTLS_PLATFORM_STD_CALLOC CHIPPlatformMemoryCalloc /**< Default allocator to use, can be undefined */
```

Comment:
Could we remove this and use mbedtls_platform_set_calloc_free to avoid the awkward dependency cycle that results from this? The platform SDK shouldn't depend on CHIP.

Reply:
OK, I'll undo this and file a separate issue to support `mbedtls_platform_set_calloc_free()` for  `chip_config_memory_management ≠ "malloc"`.

Reply:
#3178 Use mbedtls_platform_set_calloc_free() for non-malloc() configurations